### DISCUSSION TITLE
Github is my work tool, please create a home page that help me instead of taking my focus away

### DISCUSSION THREAD

**Discussion by MaelAbgrall** [community/community#53780] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

if there is a proposition to make, **I would much rather have a home screen that shows exactly where my attention should be on**.

When I log in to GitHub, this is to check my repositories, issues, PR, or anything that need my attention.

Having a social media-oriented home screen is the complete opposite of what I need and takes my focus away from what I need to do.

some ideas would be:
- Failed flows
- Propose a merge if I pushed a branch in any repository
- updates of open PR in my repos/repos I'm following
- discussions updates if I made them, or follow them
- **Anything that I need to take care of in MY repo, MY org repo, or repos I am watching, ordered by how important it is to do it**

and with the option of course to disable one or multiple of those features, should I not use them

**Comment by PranavVerma-droid** [comment]
I actually like this, and **wishing for it to be implemented** :D

**Reply by andrew-goldie** [reply]
Totally agree with OP. Never use that feed and have to spend several clicks getting to where I need to be EVERY TIME! Can it be a simple preference to choose which page is default? For me it would either be a list of my organisations because I flip between a few as a freelancer, or at least the repos of the last-visited organisation.

**Comment by slyngshede** [comment]
That would make the home screen actually useful, compared to now where it's just a speed bump on the road to where you're actually going.

The whole attempt at putting a social network into the home screen is only made worse by the fact that getting to where you need/want to go it pretty difficult. I think it's weird that there's no direct links to "My repositories", which is where I want to be for most of the time anyway.

Not really sure what type of workflow is imagined to benefit from the current home screen.

**Comment by hogqyq8Nifmepospew** [comment]
It’s about to joined, Together, European Union, Privacy Shield, Department of State, Department of Commerce, Department of Homeland Security, USCIS/DOJ/DOL/EO/FOIA/DOD/DOE/DOH/NIST/CISA etc.and for the Democracy’s of “Cubans” as a slaver country. This Project and Objectives was made in “Sam.gov” collaborations in all aspects.So let us proceed and see how it works. “We,You,& Me” In one way, with Humanity, Dignity, Freedom, Love, Family Oriented, And it is by the “LAW” about GitHub I had pas account that I left behind when ever happened. I’m hereby going back into this account and making sure that I have all the information that was needed to complete. Building a new one will take about not only itself, for those have related issues can participate or registered and certified. It’s to build a business, Individuals, Organizations,Local, State,National,or Federal Governments.this assignment or duties will help us build the infrastructure, Health Care, Education, Housing Assistance,Privacy Policy, American Dream, etc. a public service viewing. Let start the job wellness program for everyone.
